src/OpenTelemetry.Exporter.Zipkin/Implementation/ZipkinActivityConversionExtensions.cs

```diff
@@ -72,14 +72,43 @@ internal static ZipkinSpan ToZipkinSpan(this Activity activity, ZipkinEndpoint l
                 }
             }
 
-            if (tagState.StatusCode == StatusCode.Error)
+            if (activity.Status == ActivityStatusCode.Ok || activity.Status == ActivityStatusCode.Error)
```

Comment:
activity.Status != ActivityStatusCode.Unset?

Reply:
Logically the same, but I'm thinking maybe the current shape would have better readability?